src/seedu/addressbook/data/person/Name.java

```diff
@@ -11,9 +11,9 @@
  */
 public class Name {
 
-    public static final String EXAMPLE = "John Doe";
-    public static final String MESSAGE_NAME_CONSTRAINTS = "Person names should be spaces or alphabetic characters";
-    public static final String NAME_VALIDATION_REGEX = "[\\p{Alpha} ]+";
+    public static final String EXAMPLE = "John-John Doe";
+    public static final String MESSAGE_NAME_CONSTRAINTS = "Person names should be spaces, hyphens or alphabetic characters";
```

Comment:
Perhaps include the statement about how the names cannot be just hyphens and spaces? It would remind the user of the specific constraints.